tests/src/tracing/eventpipetrace/EventPipeTrace.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.IO;
+using Tracing.Tests.Common;
+using Microsoft.Diagnostics.Tracing;
+
+namespace Tracing.Tests
+{
+    class EventPipeSmoke
```

Comment:
Nit: Do you want to rename the class to match the file name?

Reply:
I'll fix that. This test is very much so incomplete